style.css

```diff
@@ -0,0 +1,157 @@
+html {
+    font-family: 'Open Sans', sans-serif;
+    height: 100%;
+}
+
+/* MAIN CONTENT */
+
+body {
+    display:flex;
+    font-size: 14px;
+    background-image: url(black-and-white-bground8.jpg);
+    background-size: 100%;
+    height: 80vh;
+    color: white;
+}
+.bottom-wrapper {
+    z-index:5;
+}
+section {
+    display:flex;
+    flex-wrap: wrap;
+}
+
+/* HEADER */
+
+.wrapper {
+    width:100%;
+    position:fixed;
+    z-index:10;
+    background:inherit;
+}
+header{
+    display:flex;
+    height: 20vh;
+    justify-content: flex-start;
+    align-items: center;
+    /* color: white; */
```

Comment:
Comments should serve purpose to help us so if it isn't helping, remove it and keep your code clean.